assets/css/styles.scss

```diff
@@ -0,0 +1,174 @@
+@import "variables";
+
+@mixin desktop {
+    @media (max-width: #{$desktop}){
+        @content;
+    }
+}
+
+@mixin tablet {
+    @media (max-width: #{$tablet}) {
+        @content;
+    }
+}
+
+@mixin mobile {
+    @media (max-width: #{$mobile}) {
+        @content;
+    }
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: $primaryFont;
+    font-size: $primaryFontSize;
+}
+
+body {
+    
+    .container {
+        height: 100vh;
```

Comment:
vh treba izbegavati jer se na mobilnim uredjajima ne ponasa dobro. Uvek pokusaj prvo sa `%` pa tek ako to ne ide...

Reply:
Jasno, hvala mnogo na savetu